### DISCUSSION TITLE
Livewire 3 Support

### DISCUSSION THREAD

**Discussion by nkback** [rappasoft/laravel-livewire-tables#1275] [Q&A] [answered]
Any chance we get Livewire 3 support soon? If no one is working on it, I could probably submit a PR in the next few days.

**Comment by rappasoft** [comment]
It's in the works but we might also wait for a stable release.

**Comment by lrljoe** [comment, marked as answer]
The vast majority of the work is done, its just rewriting some tests, and going through it all with a fine tooth comb.

However, there are a few extra features that I'm tinkering with too.
I agree with Anthony that we will wait for stable for an official v3 release, but fhere will be a preview (possibly for sponsors initially)



Sent from Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by nkback** [comment]
Thanks for the quick reply. Looking forward to it.
